src/systems/user_commands/UserCommands.cc

```diff
@@ -1366,6 +1440,72 @@ bool VisualCommand::Execute()
   return true;
 }
 
+//////////////////////////////////////////////////
+WheelSlipCommand::WheelSlipCommand(msgs::WheelSlipParametersCmd *_msg,
+    std::shared_ptr<UserCommandsInterface> &_iface)
+    : UserCommandBase(_msg, _iface)
+{
+}
+
+//////////////////////////////////////////////////
+bool WheelSlipCommand::Execute()
+{
+  auto wheelSlipMsg = dynamic_cast<const msgs::WheelSlipParametersCmd *>(
+      this->msg);
+  if (nullptr == wheelSlipMsg)
+  {
+    ignerr << "Internal error, null wheel slip message" << std::endl;
+    return false;
+  }
+
+
+  Entity modelEntity = this->iface->ecm->EntityByComponents(
+    components::Name(wheelSlipMsg->model_name()));
+  if (kNullEntity == modelEntity)
+  {
+    ignerr << "Failed to find model with name [" << wheelSlipMsg->model_name()
+           << "]." << std::endl;
+    return false;
+  }
+  Model model{modelEntity};
+  auto linkName = wheelSlipMsg->link_name();
+
+  auto doForEachLink = [this, wheelSlipMsg](Entity linkEntity) {
+    auto wheelSlipCmdComp =
+      this->iface->ecm->Component<components::WheelSlipCmd>(linkEntity);
+    if (!wheelSlipCmdComp)
+    {
+      this->iface->ecm->CreateComponent(
+          linkEntity, components::WheelSlipCmd(*wheelSlipMsg));
+    }
+    else
+    {
+      auto state = wheelSlipCmdComp->SetData(*wheelSlipMsg, this->wheelSlipEql) ?
```

Comment:
This line can be split into two to make the linter happy

Reply:
done in https://github.com/ignitionrobotics/ign-gazebo/pull/1241/commits/05cdbb950a372ff7afaa9a83ac7a65b78f691bfa